README.md

```diff
@@ -139,6 +139,7 @@ import { PSBrowseWrap } from '@pactsafe/pactsafe-react-sdk'
 |  `signerIdSelector`  | The ID of the `<input>` element that will be used to identify the signer.                                                                                                                                                                                                                                                                                                          |                                    string                                    |                     Yes                     |                Required Value                |
 |      `signerId`      | Use this to set the signer id directly. Note that if this value is tied to a state variable updated via user input, you may hit rate limits if it is updated frequently in a short period of time. To avoid hitting a rate limit, it is best to set the value tied to this prop only when the user's input is complete as opposed to changing this value on a per character basis. |                                    string                                    | No, unless `signerIdSelector` is not passed |                   undefined                  |
 |      `testMode`      | Enable this to register any contract acceptances as test data that can be cleared within the PactSafe UI                                                                                                                                                                                                                                                                           |                                     bool                                     |                      No                     |                     false                    |
+|      `allowDisagreed`      | Enable this to allow an invalid event is triggered when user uncheck the box                                                                         |                                     bool                                     |                   If `onInvalid` is passed                      |                     undefined                    |
```

Comment:
Might want to reword this to something like: "Enable this to allow invalid events to be triggered when a signer unchecks a checkbox." or something along those lines.